samples/04.2-modify-transferprocess/watchdog/src/main/java/org/eclipse/dataspaceconnector/samples/sample042/CheckTimeoutCommand.java

```diff
@@ -0,0 +1,32 @@
+package org.eclipse.dataspaceconnector.samples.sample042;
+
+import org.eclipse.dataspaceconnector.spi.transfer.command.Command;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.TransferProcessStates;
+
+import java.time.Duration;
+
+public class CheckTimeoutCommand extends Command {
```

Comment:
`CheckTransferProcessTimeoutCommand` is a more informative name what is really being handled by this command